raiden-ts/src/raiden.ts

```diff
@@ -955,6 +956,29 @@ export class Raiden {
       )
       .toPromise();
   }
+
+  /**
+   * Mints the amount of tokens of the provided token address.
+   * Throws an error if executed on main net.
+   *
+   * @param tokenAddress - Address of the token to be minted
+   * @param amount - Amount to be minted
+   * @returns transaction
+   */
+  public async mint(tokenAddress: string, amount: BigNumberish): Promise<ContractTransaction> {
+    // Check whether address is valid
+    if (!Address.is(tokenAddress)) throw new Error('Invalid address');
+
+    // Check whether we are on a test network
+    const { name } = await this.deps.provider.getNetwork();
+    if (name === 'homestead') {
```

Comment:
There exists `this.deps.network`, use it to avoid the uneeded async call here, please
```suggestion
    if (this.deps.network.name === 'homestead') {
```